dataactbroker/fileRoutes.py

```diff
@@ -260,3 +320,12 @@ def wrapped(*args, **kwargs):
                 "submission_id is required", StatusCode.CLIENT_ERROR)
         return fn(submission_id, *args, **kwargs)
     return wrapped
+
+
+def _find_existing_submissions(sess, submission_id, cgac_code, reporting_fiscal_year, reporting_fiscal_period):
+    return sess.query(Submission).filter(
```

Comment:
This should return the most recent submission (may need to be confirmed with @kaitlin) that matches this criteria. Currently it returns the first one it finds (based on running it locally).